src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetPeerCountResponseTests.cs

```diff
@@ -42,7 +42,7 @@ public void GetPeerCountResponse_Can_Get_Output()
             //Arrange
             var getPeerCountResponse = new GetPeerCountResponse {PeerCount = 50};
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
-            var getPeerCountCommand = new PeerCountCommand(commandContext, Substitute.For<ILogger>());
+            new PeerCountCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'PeerCountCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)